Annex60/Fluid/Movers/BaseClasses/FlowMachineInterface.mo

```diff
@@ -91,11 +107,19 @@ model FlowMachineInterface
         iconTransformation(extent={{100,-100},{120,-80}})));
 
   // "Shaft rotational speed";
-  Real r_N(min=0, unit="1") "Ratio N_actual/N_nominal";
+  Modelica.Blocks.Interfaces.RealOutput r_N(min=0, unit="1")
+    "Ratio N_actual/N_nominal";
   Real r_V(start=1, unit="1") "Ratio V_flow/V_flow_max";
 
  // Derivatives for cubic spline
 protected
+  final parameter Boolean preSpe = preVar == Annex60.Fluid.Types.PrescribedVariable.Speed
```

Comment:
I think preSpe is only used once. Does it really make sense to introduce a protected parameter to use it one time only?